soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -2977,7 +2977,14 @@ s32 func_808358F0(Player* this, PlayState* play) {
         AnimationContext_SetCopyAll(play, this->skelAnime.limbCount, this->upperSkelAnime.jointTable,
                                     this->skelAnime.jointTable);
     } else {
-        LinkAnimation_Update(play, &this->upperSkelAnime);
+        // #region SOH [Enhancement]
+        if (CVarGetInteger(CVAR_ENHANCEMENT("BoomerangReticle"), 0) == 1) {
+            // don't play animation
+        // #endregion
+        }
+        else {
+            LinkAnimation_Update(play, &this->upperSkelAnime);
+        }
```

Comment:
I think it's just cleaner to keep these as !CVar ... Once we establish hooks here it should be more clear how the `#region SOH` will fit in here, not entirely convinced on what it is now/with my suggestion
```suggestion
        // #region SOH [Enhancement] - Skip Playing Animation
        if (!CVarGetInteger(CVAR_ENHANCEMENT("BoomerangReticle"), 0)) {
        // #endregion
            LinkAnimation_Update(play, &this->upperSkelAnime);
        }
```